### ISSUE TITLE
Pressing "Delete" with focus on individual clip should remove them.

### ISSUE THREAD

**Issue by felagund** [ozmartian/vidcutter#54]
When focusing the individual clips on the right, I can remove them by rightclicking them and selecting "remove" in the menu. I think a keyboard shortcut linked to "Delete" would be logical here.

**Comment by Xavion** [comment]
I completely agree with this.  It is annoying having to move the mouse down, click on 'remove', and then still have to choose an option (one/all) from the list.  The "Delete" key should be used to remove the selected ones, while perhaps "Shift + Delete" could empty the list entirely.
